src/main/webapp/app/exercises/modeling/manage/modeling-exercise-update.component.ts

```diff
@@ -63,6 +64,14 @@ export class ModelingExerciseUpdateComponent implements OnInit {
         private router: Router,
     ) {}
 
+    get editType(): EditType {
+        if (this.isImport) {
+            return EditType.IMPORT;
+        }
+
+        return this.modelingExercise.id === undefined ? EditType.CREATE : EditType.UPDATE;
```

Comment:
```suggestion
        return this.modelingExercise.id == undefined ? EditType.CREATE : EditType.UPDATE;
```
(Using only `==` checks also for `null` in this particular case.)

Reply:
you are right, done.